sandboxed_containers/sandboxed-containers-4.11-release-notes.adoc

```diff
@@ -26,7 +26,78 @@ This product is fully supported and enabled by default as of {product-title} 4.1
 [id="sandboxed-containers-1-3-known-issues"]
 == Known issues
 
-
+* If you are using {sandboxed-containers-first}, you might receive SELinux denials when accessing files or directories mounted from the `hostPath` volume in an {product-title} cluster. These denials can occur even when running privileged sandboxed containers because privileged sandboxed containers do not disable SELinux checks.
++
+Following SELinux policy on the host guarantees full isolation of the host file system from the sandboxed workload by default, and provides stronger protection against potential security flaws in the `virtiofsd` daemon or QEMU.
++
+If the mounted files or directories do not have specific SELinux requirements on the host, you can use local persistent volumes as an alternative. Files are automatically relabeled to `container_file_t`, following SELinux policy for container runtimes. See xref:../storage/persistent_storage/persistent-storage-local.adoc[Persistent storage using local volumes] for more information.
++
+Automatic relabeling is not an option when mounted files or directories are expected to have specific SELinux labels on the host. Instead, you can set custom SELinux rules on the host in order to allow the `virtiofsd` daemon to access these specific labels. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1904609[*BZ#1904609*])
+
+* The {sandboxed-containers-operator} pods that use container CPU resource limits to increase the number of available CPUs for the pod might receive fewer CPUs than requested. If the functionality is available inside the container, you can diagnose CPU resources by using `oc rsh <pod>` and running the `lscpu` command.
++
+[source,terminal]
+----
+$ lscpu
+----
++
+.Example output
++
+[source,terminal]
+----
+CPU(s):                          16
+On-line CPU(s) list:             0-12,14,15
+Off-line CPU(s) list:            13
+----
++
+The list of available offline CPUs will likely change from run to run in an unpredictable manner.
++
+As a workaround, you can use a pod annotation to request additional CPUs rather than setting a CPU limit. The method of allocating processors is different, and CPUs requested by means of pod annotation are not affected by this issue. Rather than setting a CPU limit, the following `annotation` must be added to the metadata of the pod:
```

Comment:
Is the allocation method mentioned in the second line is specific to using pod annotations to request the additional CPUs? If so, you could make this clearer by reversing the structure of the second sentence. For example:

> CPU requests that use pod annotation are not affected by this issue, because the processor allocation method is different.

Reply:
@cpmeadors or @tbuskey - would this sentence change still be correct?

Reply:
I think it says the same thing and is correct.